src/doodle/doodleController.js

```diff
@@ -97,20 +109,24 @@ export class DoodleController {
     };
     render(
       <Fragment>
-        <DoodleCanvas
-          attachedElement={this._container}
-          size={this._size}
-          tool={this._tool}
-          active={this._doodleable}
-          lines={this.newLines}
-          setLines={setLines}
-          color={this._color}
-        />
-        <DisplayCanvas
-          handleDoodleClick={this._handleDoodleClick}
-          doodles={this.savedDoodles}
-          container={this._container}
-        />
+        {this._showDoodles && (
+          <Fragment>
+            <DoodleCanvas
```

Comment:
For performance reasons, it's probably better to just set CSS on the canvas/parent to make it invisible, so the canvas doesn't need to be fully redrawn when re-showing doodles.

Also, maybe still show the DoodleCanvas, as that's under their control? (Or only if it's active?) IDK, there's no existing behavior to copy for that, so it's mostly up to us.